src/bench/bls.cpp

```diff
@@ -129,7 +130,7 @@ static void BLS_Verify_LargeBlock(size_t txCount, benchmark::Bench& bench, uint3
     // Benchmark.
     bench.minEpochIterations(epoch_iters).run([&] {
         for (size_t i = 0; i < pubKeys.size(); i++) {
-            sigs[i].VerifyInsecure(pubKeys[i], msgHashes[i]);
+            assert(sigs[i].VerifyInsecure(pubKeys[i], msgHashes[i]));
```

Comment:
should have no function calls inside asserts
```suggestion
            bool ok = sigs[i].VerifyInsecure(pubKeys[i], msgHashes[i]);
            assert(ok);
```
same for other asserts below